app/workers/follows/send_email_notification_worker.rb

```diff
@@ -9,7 +9,7 @@ def perform(follow_id, mailer = NotifyMailer.name)
 
       return if EmailMessage.where(user_id: follow.followable_id)
         .where("sent_at > ?", rand(15..35).hours.ago)
-        .exists?(["subject LIKE ?", "%#{NotifyMailer.subjects[:new_follower_email]}"])
+        .exists?(["subject LIKE ?", "%#{NotifyMailer.new.subjects[:new_follower_email]}"])
```

Comment:
I unintentionally found out that this is a syntax error and this code may have been broken the whole time.

Reply:
@maestromac A syntax error? Not a `NoMethodError`? 😕

Reply:
@citizen428 ah yup I meant `NoMethodError`